docs/SPEC.md

```diff
@@ -751,6 +767,7 @@ Operator precedence is encoded directly into the grammar.
     PostfixOperator          = MemberExpression
                              | CallExpression
                              | IndexExpression .
+                             
```

Comment:
The grammar above says `MultiplicativeExpression = AdditiveExpression` that does not match with your updates below 

```
AdditiveExpression             = MultiplicativeExpression { AdditiveExpressionSuffix } .
```